.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+aliases:
+  - &install-pip |
```

Comment:
what is &?

Reply:
never mind, found it -- https://circleci.com/blog/decrease-your-build-times-by-running-jobs-in-parallel-with-workflows/